### DISCUSSION TITLE
/hyndsight/benchmarks/

### DISCUSSION THREAD

**Discussion by robjhyndman** [robjhyndman/robjhyndman.com#52] [Announcements]
Benchmarks for forecasting 
 https://robjhyndman.com/hyndsight/benchmarks/

**Comment by robjhyndman** [comment]
```header
Originally posted by Kanti Mohan Pandit on 2010-08-25 15:51:00
````
I have read your advice for authors to convince you on the new and
proposed methods for forecasting and must include comparisons with
standard benchmark. If the author is not conversant about the standard
other forecasting methods but he has invented innovative method of his
own how can he justify the comparisons. He can of course place large
scale empirical evidence to prove his theory with results. I feel your
suggestion can not be carried out in this circumstance. Kindly relax and
let tnew author prove his caliber without comparison.\
With kind Regards\
Kanti Mohan Pandit

**Reply by robjhyndman** [reply]
```header
Originally posted on 2010-08-25 23:18:57
````
Large scale empirical studies are only useful when a benchmark is
included for comparison. Otherwise how can you argue that your new
method is useful? An important part of research is being familiar with
previous literature on the subject, and that includes being familiar
with the standard methods that are used and the relevant benchmarks that
are appropriate.

**Comment by robjhyndman** [comment]
```header
Originally posted by Chewyraver on 2010-09-16 09:30:08
````
I fully agree with this, I don\'t do many comparisons, but I do always
do some, even if it is only the random walk model for the reasons you
list here.

**Comment by robjhyndman** [comment]
```header
Originally posted by Peter Ellis on 2016-11-09 21:07:17
````
Are there any published large collections of time series data with
external regressors, comparable to the M competition data but with
covariates?

**Reply by robjhyndman** [reply]
```header
Originally posted on 2016-11-09 21:21:24
````
Not that I know about, unfortunately.

**Reply by woodjmichael** [reply]
Why, in your opinion, is something like ARIMA really necessary? And are we ~~use~~ sure that authors are not, unintentionally or even intentionally, letting their ARIMA underperform to make their method look better? I real a lot of papers where even if I had the data, I couldn't reproduce their benchmark results because they haven't explained the model and training procedure well enough.

By comparison, the naive methods you suggest seem (a) incredibly transparent, (b) nearly trivial to reproduce. Those two points seems to me most important for a good benchmark, not the absolute performance of the benchmark.

**Reply by robjhyndman** [reply]
1. ARIMA models are often the best available univariate time series models. As a benchmark, they can be useful when comparing to multivariate time series models such as VARs to check that the extra effort in modelling contemporaneous correlations gives better forecasts (spoiler: it usually doesn't). 
2. Not explaining the benchmarks is a common problem. If someone is using R, then using the `auto.arima()` function from the `forecast` package, or the `ARIMA()` function from the `fable` package is a good option as it is reproducible.

**Comment by El-Ayoubi-Hub** [comment]
Can you please elaborate on the case when the new method doesn't beat the benchmarks? what other factors do you take into account (if you can provide real examples from your experience it will be very welcome)?
